tests/testthat/test-get_poly_attribute.R

```diff
@@ -14,11 +14,11 @@ if (suppressPackageStartupMessages(require("sp"))) {
   unioned_spdf <- self_union(spdf)
 
   test_that("works with numbers", {
-    expect_equivalent(get_poly_attribute(unioned_spdf$union_df, "a", sum), c(1,2,3))
+    expect_equivalent(sort(get_poly_attribute(unioned_spdf$union_df, "a", sum)), c(1,2,3))
```

Comment:
I did not know about `expect_seteqal()`! I think that's exactly what we need here, then we don't need to worry about sorting

Reply:
Done!